dot/core/runtime.go

```diff
@@ -17,18 +17,28 @@
 package core
 
 import (
+	"bytes"
 	"errors"
 	"fmt"
 
 	"github.com/ChainSafe/gossamer/dot/core/types"
 	"github.com/ChainSafe/gossamer/lib/babe"
+	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/runtime"
 	"github.com/ChainSafe/gossamer/lib/scale"
 	"github.com/ChainSafe/gossamer/lib/transaction"
 
 	log "github.com/ChainSafe/log15"
 )
 
+// StorageRoot returns the hash of the runtime storage root
+func (s *Service) StorageRoot() (common.Hash, error) {
```

Comment:
this was just moved to `runtime` since it relates to runtime

Reply:
this isn't really related to runtime, not sure if it belongs here

Reply:
"returns the hash of the runtime storage root" ?

Reply:
yeah that's not really correct it should just say storage root

Reply:
ok, will move back to `service.go` and update comment